dev-python/black/black-19.10_beta_p0.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+MY_PV="${PV/_beta/b}"
+MY_PV="${MY_PV/_p/}"
+
+DESCRIPTION="The uncompromising Python code formatter"
+HOMEPAGE="https://github.com/psf/black"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${PN}-${MY_PV}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
```

Comment:
Do you need a RESTRICT for this?

Reply:
In what case would I need that?

Reply:
You only want to run tests if USE=test, right?

Reply:
no, tests are triggered by `FEATURES=test`

Reply:
@thesamesam was right. You need
```bash
RESTRICT="!test? ( test )"
```
Please see https://devmanual.gentoo.org/general-concepts/dependencies/#test-dependencies for more information.